src/visitor.rs

```diff
@@ -432,6 +433,7 @@ impl<'a> FmtVisitor<'a> {
                         &item.vis,
                         body,
                     ),
+                    generics,
```

Comment:
If you expand the code above, you'll notice that generics is now being passed twice. Once as part of the ItemFn creation, and once as part of visit_fn. This redundancy is further reflected in `visit_fn` where the new `generics` argument is not actually used in the `ItemFn` variant of the match statement. (@nikomatsakis this is what I mentioned to you over chat)